### PR TITLE
Initial changes to support Outcome Architectures

### PR DESCRIPTION
**Pull request by craigtracey**
Initial changes to support Outcome Architectures

- Add styling from the Figma design
- Add some initial content to demonstrate the outcome styling

### CODE REVIEW

**Comment by bryanfriedman** [comment]
@calebwashburn @zmb3 How does this PR jive with the ["series" PR](https://github.com/vmware-tanzu/tanzu-dev-portal/pull/664) that @johnharris85 has been working on? Is this separate or related somehow? /cc @alexanderstran @BrianMMcClain

File: custom_dict.txt [hunk 1]
```diff
@@ -660,6 +662,7 @@ LimitRanges
 LinkedIn
 Linkerd
 linter
```
Comment (zmb3) [inline custom_dict.txt:664]: We can remove `linter` since it's covered by `linter(s)`

File: custom_dict.txt [hunk 2]
```diff
@@ -1054,7 +1059,10 @@ Siri
 Skyscanner
 SLA
 Sletten
+SLI(s)
 slideshare
+SLO
```
Comment (zmb3) [inline custom_dict.txt:1064]: Remove `SLO`.

File: custom_dict.txt (hunk 1 above)
Comment (craigtracey) [reply custom_dict.txt:664]: The check was failing on `linter` specifically

**Review by BrianMMcClain — APPROVED** [review]

**Pull request merged**